docs/static/configuration.asciidoc

```diff
@@ -11,7 +11,7 @@ Let's step through creating a simple config file and using it to run Logstash. C
 ----------------------------------
 input { stdin { } }
 output {
-  elasticsearch { hosts => ["localhost:9200"] }
+  elasticsearch { hosts => ["http://localhost:9200"] }
```

Comment:
`elasticsearch { hosts => ["localhost:9200"] }` is also valid, see https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-hosts